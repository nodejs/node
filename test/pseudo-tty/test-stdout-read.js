'use strict';
const common = require('../common');
process.stderr.on('data', common.mustCall(console.log));
