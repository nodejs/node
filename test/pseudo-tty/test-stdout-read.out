Hello!
<Buffer 48 65 6c 6c 6f 21 0a>
